ui/src/app/applications/components/pod-logs-viewer/pod-logs-viewer.scss

```diff
@@ -156,14 +158,19 @@
 
 .copySuccess {
     // ARGO_SUCCESS_COLOR
-    background-color: #18BE94 !important;
+    background-color: #18be94 !important;
 }
 .copyFailure {
     // ARGO_FAILED_COLOR
-    background-color: #E96D76 !important;
+    background-color: #e96d76 !important;
```

Comment:
Since we're importing the config styles here now, can we switch these to the SASS variables (e.g. `$argo-failed-color`)?

Reply:
Thanks for the review. This is automatically changed by the formatter when I save the file which I didn't pay much attention on. Do you mean I should fix only here or all hex codes in this scss file? Or better I discard this change and open another issue to update all hex code into SASS variables?

Uploaded a screenshot in the first comment. Let me know if it looks good, thanks.

Reply:
I see, I think we can leave the hex codes for now in that case